modules/multiplayer/scene_replication_interface.h

```diff
@@ -33,33 +33,77 @@
 
 #include "scene/main/multiplayer_api.h"
 
-#include "scene_replication_state.h"
+#include "modules/multiplayer/multiplayer_spawner.h"
+#include "modules/multiplayer/multiplayer_synchronizer.h"
```

Comment:
Use relative includes for module headers so it doesn't break if the module is renamed / used from out of source via `custom.py`/`custom_modules` SCons option.